core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/Parameter.kt

```diff
@@ -1,14 +1,15 @@
 package ch.ergon.dope.resolvable.expression.unaliased.type
 
 import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.DopeQueryManager
 import ch.ergon.dope.resolvable.expression.TypeExpression
 import ch.ergon.dope.validtype.ArrayType
 import ch.ergon.dope.validtype.BooleanType
 import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-object ParameterManager {
+class ParameterManager {
     var count: Int = 1
         get() = field++
 
```

Comment:
please remove the _resetCounter()_-method, I think it's not needed anymore

Reply:
and please remove it in the other manager as well

Reply:
done. I also made the setter private that the user can't set the counter